src/Inkluzitron/Services/TypeReaders/BooleanTypeReader.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Inkluzitron.Services.TypeReaders
+{
+    public class BooleanTypeReader : TypeReader
+    {
+        private Dictionary<Func<Regex>, bool> MatchingFunctions { get; } = new Dictionary<Func<Regex>, bool>()
+        {
+            { () => new Regex("^(ano|yes|true?)$"), true }, // ano, yes, true, tru

```

Comment:
Lambda tu nema vyznam a navic zpusobuje, ze se regex vytvari znovu pri kazdem nacitani booleanu.
I ten regex je docela kanon na vrabce, ale OK, pokud se nebude vytvaret pokazde novy.

Reply:
Upravil jsem to, aby se to načítalo jen jednou.